templates/raspbian/resources/usr/share/initramfs-tools/scripts/cattlepi-base/helpers

```diff
@@ -140,37 +140,47 @@ cattlepi_run_bootcode()
 
 cattlepi_fetch_update_images()
 {
+    # Read the md5's of the current init/rootfs images for the purpose
+    # of removing them after updates
+    if [[ -r "${CATTLEPI_DIR}/config" ]]; then
+        local cur_initfs_md5=$(jq '.initfs.md5sum' "${CATTLEPI_DIR}/config")
```

Comment:
this assumes that there is a config file in there and that this code is the only one that touches/updates the file.
what happens if the file is not there or the file gets updated through other mechanisms (not the case today, but I can see it happening in the future)

Reply:
If the config file doesn't exist, the function should continue on and pull down the config.  My changes, however, don't cover this case and will leak images if the config file gets deleted somehow.

I'm curious:
* what will change for this code if the file is updated through other mechanisms?
* what other mechanisms are on your mind?